crypto/rand/rand_unix.c

```diff
@@ -450,14 +461,14 @@ void rand_pool_keep_random_devices_open(int keep)
  */
 size_t rand_pool_acquire_entropy(RAND_POOL *pool)
 {
-#  ifdef OPENSSL_RAND_SEED_NONE
+#  if defined(OPENSSL_RAND_SEED_NONE)
```

Comment:
The `#ifdef OPENSSL_RAND_SEED_XXX` and `#if defined(OPENSSL_RAND_SEED_XXX)` conditions were mixed at random. I chose the latter for all.